packages/contracts/contracts/votings/whitelist/WhitelistVoting.sol

```diff
@@ -26,8 +26,9 @@ contract WhitelistVoting is MajorityVoting {
 
     error VoteCreationForbidden(address sender);
 
-    event AddUsers(address[] users);
-    event RemoveUsers(address[] users);
+    event UsersAdded(address[] users);
+
+    event UsersRemoved(address[] users);
```

Comment:
do we need to add NatSpec here?
if yes, the re-visit this file for events and errors